crates/forge/bin/cmd/build.rs

```diff
@@ -66,6 +66,12 @@ pub struct BuildArgs {
     #[serde(skip)]
     pub ignore_eip_3860: bool,
 
+    /// Detect missing libraries, instead of erroring
+    ///
+    /// Currently unused
+    #[clap(long = "zk-detect-missing-libraries", group = "build")]
+    pub detect_missing_libraries: bool,
```

Comment:
```suggestion
    pub zk_detect_missing_libraries: bool,
```
Since this is a foundry struct that we are adding a zksync specific value to, we should prefix with `zk`. Also maybe move to the bottom of the struct definition to separate this from the upstream fields.

Reply:
I agree about the naming, but if we change this here, we will affect existing projects currently using `detect_missing_libraries`... right?

Reply:
For those using the command line no because the value is with the `zk` prefix already. Also, this is the name of the struct field, not the user facing option for setting the flag. If we wanted to keep the non-prefixed value (don't think we should tbh), we can add an alias via the clap macro.